README.md

```diff
@@ -22,10 +24,48 @@
 
 # react-rails <a name="react-rails"></a>
 
-> This is just an Api for a react app here is the link of the react app => [react-rails](https://github.com/Mov305/rails_react_front)
+> This is just an Api for a react app here is the link of the react app => [react-rails](https://github.com/zdnahom/book-a-room-frontend)
```

Comment:
- [ ] OPTIONAL: On the frontend project, In your `App` refactoring, a linter problem was introduced:

![image](https://github.com/Mov305/book_a_room_backend/assets/57421823/9216da31-8752-48ea-afd0-7a67957e1093)

![image](https://github.com/Mov305/book_a_room_backend/assets/57421823/e6a9c72d-4129-48a0-bdc6-e513ca3c1505)

This is because the return statement is the whole function, so no brackets are needed.  Also, there is no need for a return statement anymore.

This is the way to correct this issue:

![image](https://github.com/Mov305/book_a_room_backend/assets/57421823/3c5beb73-0f7d-4803-a3ad-c40f1a398f62)

Notice that since the whole function is the return statement, the brackets have been removed, and the `return `statement has been replaced by just a parenthesis. 👍

Reply:
Thank you for the review